core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -341,15 +353,28 @@ oppia.controller('ExplorationEditor', [
     };
 
     $scope.showWelcomeExplorationModal = function() {
+      editorFirstTimeEventsService.initRegisterEvents($scope.explorationId);
```

Comment:
What if the user has created some other exploration before? In that case they wouldn't be shown this modal, I think....